test-workflows/task/eks-cluster-teardown/eksctl/0.1/delete-eks-cluster.yaml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Let's talk about the organizational structure of the test-workflow tasks (I missed this last time). IMO, it's pretty (perhaps overly) verbose. 

Thoughts on something like 

```
tests/tasks
tests/tasks/setup/eks-xlarge.yaml
tests/tasks/setup/kit-medium.yaml
tests/tasks/setup/kit-large.yaml
tests/tasks/setup/kit-xlarge.yaml
tests/tasks/load-generators/
tests/tasks/teardown/eks.yaml
tests/tasks/teardown/kit.yaml
...
tests/pipelines/eks-large-networking
tests/pipelines/eks-large-pod-churn
...
```

Reply:
Sure, we can discuss offline. Part of the reason for being overly verbose is because we have different implemetations of creating K8s/eks clusters. Using, akt, eksctl, kit etc. We might need a way to differentiate them well.

Reply:
Offline discussion. One option could be:
`tests/tasks/setup/eksctl/xlarge.yaml`
`tests/tasks/teardown/eksctl.yaml`
`tests/tasks/generator/clusterloader/networking.yaml`